### ISSUE TITLE
Release 0.7.0

### ISSUE THREAD

**Issue by JamesGuthrie** [timescale/promscale_extension#527]
## Pre-release
- [x] Create a git branch named `pre-release-x.x.x`
- [x] Ensure `upgradeable_from` in `templates/promscale.control` contains the previously released version.
- [x] Update `CHANGELOG.md` with release version and date, ensuring that all relevant changes are reflected.
- [x] Update the version in all places with `./update-version.sh <version-here>`
- [x] Freeze any new incremental sql scripts released in this version in `incremental_freeze_test`
- [x] Create a PR from the `pre-release-x.x.x` branch. Get it approved. Merge the PR to master.

## Release
- [x] Create a tag on the master branch `git tag <new version>`
- [x] Push the tag `git push origin <new version>`
- [x] CI will trigger and create a draft release with assets attached.
- [x] Prepare extension release notes, share with team asking for feedback.
- [x] Attach release notes to draft release created above.
- [x] Wait for CI to generate the packages files
- [x] Create a PR to update the HA image in [timescaledb-docker-ha](https://github.com/timescale/timescaledb-docker-ha). [EXAMPLE](https://github.com/timescale/timescaledb-docker-ha/pull/285/files)
- [x] In the timescaledb-docker-ha repo, update `TIMESCALE_PROMSCALE_EXTENSIONS` in the `Makefile` to include the version just released
- [x] Update the CHANGELOG entry in the timescaledb-docker-ha repo and wait for the CI to complete and request review from the Cloud team and merge it when approved.
- [x] Create a new PR in the timescaledb-docker-ha repo. Stamp the version in the CHANGELOG. Merge it with master and push the correct tag to trigger CI ([see instructions in the repo](https://github.com/timescale/timescaledb-docker-ha#release-process)) [EXAMPLE](https://github.com/timescale/timescaledb-docker-ha/pull/286/files)
- [x] Publish the GitHub release on the promscale_extension repo.

## Post-Release
- [x] Create a new git branch named `post-release-x.x.x`
- [x] Run `make post-release` which will generate the `sql/promscale--x.x.x.sql` file of the version just released and create all the upgrade path sql files.
- [x] Add and commit the newly created sql files to git. They are ignored by default. e.g. `git add sql/*--0.5.5.sql --force`
- [x] Determine the development version (determined by bumping the patch version and appending `-dev`)
- [x] Set the version in all places necessary with `./update-version.sh <develop-version>`
- [x] Update `upgradeable_from` in templates/promscale.control to add the previously released version
- [ ] Update `e2e/tests/config/mod.rs` to refer to the new docker images
- [ ] Create a PR and get it merged
- [ ] Bump the version in the promscale repo's `EXTENSION_VERSION` file to the version just released (Renovate should automatically create a PR for this).

**Comment by JamesGuthrie** [comment]
Release is currently blocked on compatibility with TimescaleDB 2.8, tracked in: 
https://github.com/timescale/promscale_extension/issues/502.
